hardware_translator:
  ros__parameters:
    prefix: "esp32"
    publish_rate: 50.0
    joint_names: [
      "front_left_wheel_joint",
      "front_right_wheel_joint",
      "rear_left_wheel_joint",
      "rear_right_wheel_joint"
    ]